workos/user_management.py

```diff
@@ -444,6 +446,9 @@ def get_authorization_url(
                 OktaSAML, and AzureSAML connection types. (Optional)
             state (str) - An encoded string passed to WorkOS that'd be preserved through the authentication workflow, passed
                 back as a query parameter. (Optional)
+            code_challenge (str) - Code challenge is derived from the code verifier used for the PKCE flow. (Optional)
+            code_challenge_method ("S256") - The only valid PKCE code challenge method is "S256".
```

Comment:
if there's only 1 option for this field, why pass it in at all? let's just set it only when code_challenge is passed in

Reply:
Updated ✅